db/schema.sql

```diff
@@ -1,3 +1,16 @@
+drop table if exists Users cascade;
```

Comment:
Don't think you need to drop table one by one, since shengmin already added
`DROP SCHEMA public CASCADE;`
`CREATE SCHEMA public;`